src/ui/widgets/editors/w_RoutesEditor.cpp

```diff
@@ -299,9 +299,14 @@ CONFIGROOT RouteEditor::OpenEditor()
     {
         // Process Browser Forwarder
         QJsonObject browserForwarder;
-        browserForwarder["listenAddr"] = bfListenIPTxt->text();
-        browserForwarder["listenPort"] = bfListenPortTxt->value();
-        root["browserForwarder"] = browserForwarder;
+		if(bfListenIPTxt->text().trimmed() == ""){
```

Comment:
I remembered that there is a method called isEmpty on QString...

Reply:
😅😅😅